# Ensure that emacs uses GNU find. See components/emacs/README.
#    16171813 emacs rgrep assumes gnu find in path; fails using
#            GNU-specific options
#
--- emacs-29.3.orig/lisp/ldefs-boot.el	2024-03-24 07:34:55.000000000 -0600
+++ emacs-29.3/lisp/ldefs-boot.el	2024-03-25 13:29:17.232073269 -0600
@@ -14726,7 +14726,7 @@
 (defvar grep-program (purecopy "grep") "\
 The default grep program for `grep-command' and `grep-find-command'.
 This variable's value takes effect when `grep-compute-defaults' is called.")
-(defvar find-program (purecopy "find") "\
+(defvar find-program (purecopy "/usr/gnu/bin/find") "\
 The default find program.
 This is used by commands like `grep-find-command', `find-dired'
 and others.")
--- emacs-29.3.orig/lisp/loaddefs.el	2024-03-24 07:23:19.000000000 -0600
+++ emacs-29.3/lisp/loaddefs.el	2024-03-25 13:29:17.236491647 -0600
@@ -14726,7 +14726,7 @@
 (defvar grep-program (purecopy "grep") "\
 The default grep program for `grep-command' and `grep-find-command'.
 This variable's value takes effect when `grep-compute-defaults' is called.")
-(defvar find-program (purecopy "find") "\
+(defvar find-program (purecopy "/usr/gnu/bin/find") "\
 The default find program.
 This is used by commands like `grep-find-command', `find-dired'
 and others.")
--- emacs-29.3.orig/lisp/progmodes/grep.el	2024-01-06 05:56:30.000000000 -0700
+++ emacs-29.3/lisp/progmodes/grep.el	2024-03-25 13:29:17.237129046 -0600
@@ -506,7 +506,7 @@
 This variable's value takes effect when `grep-compute-defaults' is called.")
 
 ;;;###autoload
-(defvar find-program (purecopy "find")
+(defvar find-program (purecopy "/usr/gnu/bin/find")
   "The default find program.
 This is used by commands like `grep-find-command', `find-dired'
 and others.")
